pkgs/development/python-modules/django-maintenance-mode/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchPypi, buildPythonPackage,
+  mock, django }:
+buildPythonPackage rec {
+  pname = "django-maintenance-mode";
+  version = "0.13.1";
```

Comment:
According to https://pypi.org/project/django-maintenance-mode/ there is a 0.13.3 version available. Any specific reason not to use it?

Reply:
Well, not really, I am using it with an older version of django on one of my private projects... I'll bump it, sorry!

Reply:
I sympathize with being stuck on older versions of e.g. django - is dropping support for older djangos a deliberate move in 0.13.3 by the project authors? If not, do let them know it's not working for you.